src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final String ID = "id";
+    private static final String NAME = "name";
+    private static final String COUNTRY = "country";
+    private static final String CREATE_MANUFACTURER = "INSERT INTO manufacturers (name, country)"
+            + " VALUES (?, ?)";
+    private static final String GET_MANUFACTURER_BY_ID = "SELECT * FROM manufacturers "
+            + "WHERE id = (?) AND is_deleted = FALSE;";

```

Comment:
In **read**, **update** and **delete** queries the braces near question mark are redundant
```suggestion
            + "WHERE id = ? AND is_deleted = FALSE;";
```
Braces needed only for INSERT query.
Please fix other cases in the code.